docs/_layouts/default.html

```diff
@@ -76,7 +75,7 @@
 
         <div class="buttons-unit">
           <a href="/react/docs/getting-started.html" class="button">Get Started</a>
-          <a href="/react/downloads.html" class="button">Download React v{{site.react_version}}</a>
+          <a href="/react/docs/tutorial.html" class="button">Take the Tutorial</a>
```

Comment:
Maybe a nit from a non-native speaker, but this:

<img width="435" alt="screen shot 2016-10-04 at 13 34 00" src="https://cloud.githubusercontent.com/assets/810438/19074251/433e5e30-8a37-11e6-8410-2627ff64e2eb.png">

Reminds me of [this](https://youtu.be/2dgmgub8mHw?t=22s).

I honestly wouldn’t know which to choose. If I’m new to React, should I “Get Started” or “Take the Tutorial”? Does taking a tutorial presume I already “got started”? Does “getting started” presume I already know things in the tutorial?